.paparazzirc

```diff
@@ -0,0 +1,32 @@
+{
+	"routes": [
+		"uk",
+		"showcase",
+		"uk/subscribe",
+		"uk/subscribe/paper",
+		"uk/subscribe/digital",
+		"uk/subscribe/weekly",
+		"uk/contribute",
+		"uk/contribute/recurring",
```

Comment:
I would probably ditch this one since we'll be removing the old payment flow soon - I noticed you've released a new bundle page (maybe you're A/B testing it right now?) which means these routes will soon be unreachable through any flow

Reply:
I added it especially because of the old payment flow still being partially there! it's one of the few I'm afraid of accidentally fussing up this week as the header work progresses